### ISSUE TITLE
v2: Port commandline tool

### ISSUE THREAD

**Issue by Ambrevar** [russross/blackfriday#312]
As discussed in #218 , we should port blackfriday-tool to v2.
A nit to discuss: shall we move it to blackfriday/cmd or not? I'd say yes.

**Comment by shurcooL** [comment]
> I'd say yes.

Do you have any comments on why you prefer that?

**Comment by Ambrevar** [comment]
- It's low maintainance anyways.
- One repo less to maintain.
- Psychological drive: It forces us to keep the demo tool up-to-date with the lib.
- Bikeshed ahead, let's just pick one option.

**Comment by shurcooL** [comment]
Sounds good to me.

To clarify, would its import path be:

```
gopkg.in/russross/blackfriday/cmd/blackfriday-tool
```

Is that right?

**Comment by russross** [comment]
What about

> gopkg.in/russross/blackfriday.v2/cmd/blackfriday

instead? The only reason I put the -tool suffix on there when I split it off initially was to get a unique repository name.

**Comment by Ambrevar** [comment]
Agreed with @russross : forgot to name one advantage I mentioned before, the tool can have a straightforward name now.
